### DISCUSSION TITLE
View all errors in CWD

### DISCUSSION THREAD

**Discussion by itzcodex24** [LazyVim/LazyVim#2028] [Q&A]
Hi there, is there a way to view all errors within the current working directory?

**Comment by dpetka2001** [comment]
`Telescope diagnostics` which is by default bound to `<leader>sD` shows you diagnostics for the LSP workspace. If you want only for the CWD which might be different from the LSP `root_dir`, then I believe you could do `:Telescope diagnostics root_dir=true`. Test it to see if it's what you want.

**Comment by dimas-cyriaco** [comment]
Just as an aside: not all LSP servers have project-wise diagnostics. The Typescript LSP doesn't, for example. It only knows about open files.

I use this plugin so get something similar to workspace diagnostics in Typescript: [dmmulroy/tsc.nvim](https://github.com/dmmulroy/tsc.nvim)

**Comment by erlangparasu** [comment]
`<leader>xx`  for showing **Diagnostics (Trouble)** window

Ref: https://www.lazyvim.org/keymaps#troublenvim
